js/client-mainmenu.js

```diff
@@ -1101,33 +1100,101 @@
 			} else if (teams.length > 27) {
 				bufBoundary = Math.ceil(teams.length / 2);
 			}
-
+			this.folderExpanded = data.folderExpanded || [];
+			if (data.folderToggleOn === undefined) {
```

Comment:
this seems error prone. if you want this new field to default to true, make it `dataFolderOff` and then just do `!!dataFolderOff`

Reply:
i just took it off completely, the other variables dont have this either. just make sure we declare it every time.